build.gradle

```diff
@@ -29,6 +29,23 @@ allprojects {
     }
 
     version gitVersion()
+
+    if (System.env.CIRCLE_ARTIFACTS) {
+        task copyCiArtifactsToCircle {
+            outputs.upToDateWhen { false }
+            doLast {
+                [
+                        file("auth-tokens/build/reports"),
```

Comment:
we shouldn't hardcode this -- let's use `${buildDir}` (that will apply to all projects and we won't have to have two lines here)